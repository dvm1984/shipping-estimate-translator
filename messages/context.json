{
  "shippingEstimate-bd": "Shipping estimates in business days",
  "shippingEstimate-d": "Shipping estimates in days",
  "shippingEstimate-h": "Shipping estimates in hours",
  "shippingEstimate-m": "Shipping estimates in minutes",
  "shippingEstimate-scheduled": "Shipping delivery window",
  "shippingEstimate-scheduled-no-dates": "Scheduled",
  "shippingEstimatePickup-bd": "Store pickup estimates in business days",
  "shippingEstimatePickup-d": "Store pickup estimates in days",
  "shippingEstimatePickup-h": "Store pickup estimates in hours",
  "shippingEstimatePickup-m": "Store pickup estimates in minutes"
}
