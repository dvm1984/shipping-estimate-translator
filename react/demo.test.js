import Demo from './demo'
import TestRenderer from 'react-test-renderer'
import React from 'react'
import { IntlProvider } from 'react-intl'
import messages from '../messages/pt.json'

describe('Demo', () => {
  it('Save demo strings', () => {
    const wrapper = TestRenderer.create(
      <IntlProvider locale="pt" messages={messages}>
        <Demo />
      </IntlProvider>
    )

    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
