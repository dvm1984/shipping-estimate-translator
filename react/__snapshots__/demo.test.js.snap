// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo Save demo strings 1`] = `
Array [
  <p>
    Minutos
  </p>,
  <p>
    Em alguns instantes
  </p>,
  <p>
    Em até 1 minuto
  </p>,
  <p>
    Em até 3 minutos
  </p>,
  <br />,
  <p>
    Horas
  </p>,
  <p>
    Em alguns instantes
  </p>,
  <p>
    Em até 1 hora
  </p>,
  <p>
    Em até 3 horas
  </p>,
  <br />,
  <p>
    Dias
  </p>,
  <p>
    No mesmo dia
  </p>,
  <p>
    Em até 1 dia
  </p>,
  <p>
    Em até 3 dias
  </p>,
  <br />,
  <p>
    Dias Úteis
  </p>,
  <p>
    No mesmo dia
  </p>,
  <p>
    Em até 1 dia útil
  </p>,
  <p>
    Em até 3 dias úteis
  </p>,
  <br />,
  <p>
    Pickup Minutos
  </p>,
  <p>
    Em alguns instantes
  </p>,
  <p>
    Pronto em até 1 minuto
  </p>,
  <p>
    Pronto em até 3 minutos
  </p>,
  <br />,
  <p>
    Pickup Horas
  </p>,
  <p>
    Em alguns instantes
  </p>,
  <p>
    Pronto em até 1 hora
  </p>,
  <p>
    A partir de 3 horas
  </p>,
  <p>
    Pickup Dias
  </p>,
  <p>
    Pronto no mesmo dia
  </p>,
  <p>
    Pronto em até 1 dia
  </p>,
  <p>
    Pronto em até 3 dias
  </p>,
  <br />,
  <p>
    Pickup Dias Úteis
  </p>,
  <p>
    Pronto no mesmo dia
  </p>,
  <p>
    Pronto em até 1 dia útil
  </p>,
  <p>
    Pronto em até 3 dias úteis
  </p>,
  <br />,
]
`;
